include/mbedtls/check_config.h

```diff
@@ -768,8 +768,8 @@
 #if !defined(MBEDTLS_SHA512_C)
 #error "MBEDTLS_SHA512_USE_A64_CRYPTO_* defined without MBEDTLS_SHA512_C"
 #endif
-#if defined(MBEDTLS_SHA512_ALT) || defined(MBEDTLS_SHA512_PROCESS_ALT)
-#error "MBEDTLS_SHA512_*ALT can't be used with MBEDTLS_SHA512_USE_A64_CRYPTO_*"
+#if defined(MBEDTLS_SHA512_PROCESS_ALT)
+#error "MBEDTLS_SHA512_PROCESS_ALT can't be used with MBEDTLS_SHA512_USE_A64_CRYPTO_*"
```

Comment:
Should `MBEDTLS_SHA512_PROCESS_ALT` still exist?

Reply:
No it shouldn't. This is the removal I mentioned to you where I am going to get this option, and the remaining one like it, in one commit at the end. I only discovered this type of config option when I was halfway through my initial removals.

Reply:
Still got probably 5 or so commits to add before I consider this PR done to a first approx.